dev-utils/src/commands/init_command.rs

```diff
@@ -0,0 +1,225 @@
+use super::genesis_command::GenesisCommand;
+
+use failure::Error;
+
+use super::genesis_dev_config::GenesisDevConfig;
+use read_input::prelude::*;
+use serde_json::json;
+use serde_json::Value as JsonValue;
+use std::fs;
+use std::fs::File;
+use std::io::prelude::*;
+use std::path::Path;
+use std::process::Command;
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+pub struct InitCommand {
+    data: String,
+    genesis_time: JsonValue,
+    app_hash: JsonValue,
+    app_state: JsonValue,
```

Comment:
Any specific reason for this not to be an object of `InitConfig`?

Reply:
ok, i'll change to string, DateTime<Utc>

Reply:
data= GenesisDevConfig class

Reply:
also, it reads tendermint genesis json and replace
app_state, app_hash, genesis_time